src/libsystemd/sd-bus/test-bus-cleanup.c

```diff
@@ -37,7 +37,7 @@ static int test_bus_open(void) {
         _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
         int r;
 
-        r = sd_bus_open_system(&bus);
+        r = sd_bus_open_user(&bus);
```

Comment:
Would it be better to try to connect to the system bus if an attempt to connect to the user bus fails?